share/jupyterhub/static/js/home.js

```diff
@@ -103,7 +103,7 @@ require(["jquery", "moment", "jhapi", "utils"], function(
   $(".new-server-btn").click(function() {
     var row = getRow($(this));
     var serverName = row.find(".new-server-name").val();
-    window.location.href = "../spawn/" + user + "/" + serverName;
+    window.location.href = "../spawn/" + escape(user) + "/" + serverName;
```

Comment:
@minrk did you want this to be encodeURIComponent instead of escape?

Reply:
My bad, this introduced errors, it was a bad attempt.